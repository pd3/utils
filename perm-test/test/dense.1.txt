
perm-test -t test/dense.1.tgt.txt -b test/dense.1.bg.txt -f test/dense.1.fai -c test/dense.1.calls.txt -n 1e9

             10        20        30        40        50
    12345678901234567890123456789012345678901234567890123456
       bbb bbbb bbb     ttt   bbb   ttt ttt    bbbb tttt bbb
                    ccccccc
    ....... ....... ....ooo ....... ooo.ooo ....... xxxx...
     ....... ....... ...ooo. ....... oo.ooo. ....... xxx....
      ....... ....... ..ooo.. ......x ..ooo.. ......x
       ....... ....... .ooo... .....xx .ooo... .....xx
        ....... ....... xxx.... ....xx. ooo.... ....xxx
         ....... ....... xx..... ...ooo. xx..... ...xxxx
          ....... ......x x...... ..ooo.o x...... ..xxxx.
           ....... .....oo ....... .ooo.oo ....... .xxxx..

    exact: 30./50 = 0.6
    perm-test:      0.600010167
    err:   1.0167e-05

perm-test -t test/dense.1.tgt.txt -b test/dense.1.bg.txt -f test/dense.1.fai -c test/dense.1.calls.txt -n 1e9 --no-bg-overlap

    exact: 13./50 = 0.26
    perm-test:      0.259993827
    err:   6.173e-06


