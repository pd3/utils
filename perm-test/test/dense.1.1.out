BG	1	4	6
BG	1	8	11
BG	1	13	15
TGT	1	21	23
BG	1	27	29
TGT	1	33	35
TGT	1	37	39
BG	1	44	47
TGT	1	49	52
BG	1	54	56
CALL	TGT	1	17	23
